_playwright-tests/readme.md

```diff
@@ -0,0 +1,15 @@
+# Install NPM packages
+
+yarn install
+
+# Install Playwright browsers and dependencies
+
+yarn playwright install --with-deps
+
```

Comment:
this won't work on fedora, but   `yarn playwright install` does.  Can we just say that?